src/Psalm/Internal/Stubs/CoreGenericClasses.php

```diff
@@ -597,7 +597,7 @@ public function ksort() { }
     /**
      * User defined sort
      * @link http://php.net/manual/en/arrayiterator.uasort.php
-     * @param string $cmp_function <p>
+     * @param callback $cmp_function <p>
```

Comment:
Thanks - these should instead be `callable`

Reply:
Ok, I'll change them. I used `callback` because that's what's used elsewhere in the same file.